src/MountedTraversal.js

```diff
@@ -50,6 +50,9 @@ export function instHasClassName(inst, className) {
     return false;
   }
   let classes = findDOMNode(inst).className || '';
+  if (typeof classes !== 'string') {
+    return false;
```

Comment:
This would just cause SVGs to silently fail, I don't think that's what we want. What we should do is branch on `classList` and use the old code path when it doesn't exist.

```js
export function instHasClassName(inst, className) {
  if (!isDOMComponent(inst)) {
    return false;
  }
  const node = findDOMNode(inst);
  if (node.classList) {
    return node.classList.contains(className)
  }
  let classes = node.className || '';
  if (typeof classes === 'object') {
    return classes.baseVal;
  }
  classes = classes.replace(/\s/g, ' ');
  return ` ${classes} `.indexOf(` ${className} `) > -1;
}
```